app/utils/permission.ts

```diff
@@ -65,6 +65,21 @@ const getStoragePermissionDeniedMessage = (intl: typeof intlShape) => {
     };
 };
 
+const getMicrophonePermissionDeniedMessage = (intl: typeof intlShape) => {
+    const {formatMessage} = intl;
+    const applicationName = DeviceInfo.getApplicationName();
+    return {
+        title: formatMessage({
+            id: 'mobile.microphone_permission_denied_title',
+            defaultMessage: '{applicationName} would like to access your microphone',
+        }, {applicationName}),
+        text: formatMessage({
+            id: 'mobile.microphone_permission_denied_description',
+            defaultMessage: 'Send your voice to calls participants. Open Settings to grant Mattermost access to your microphone.',
```

Comment:
@justinegeffen Can you review this message and make any suggestions please? Thanks!